src/main/kotlin/lotto/view/OutputView.kt

```diff
@@ -8,21 +8,30 @@ object OutputView {
     fun printLottos(lottos: List<Lotto>) {
         println("You have purchased ${lottos.size} tickets.")
         lottos.forEach {
-            println("[${it.rawNumbers.joinToString(", ")}]")
+            println("[${it.rawNumbers.joinToString()}]")
         }
     }
 
     fun printWinningStatistics(lottery: Lottery) {
         println("\nWinning Statistics\n------------------")
         lottery.result.entries.sortedBy { it.key.value }
             .filterNot { it.key == Prize.NONE }
-            .forEach { (prize, count) ->
-                println("${prize.matchCount} Matches (${prize.value.toMoneyExpression()} KRW) - $count tickets")
-            }
+            .forEach { (prize, count) -> printPrize(prize, count) }
 
         println("Total return rate is %.2f (A rate below 1 means a loss)".format(lottery.returnRate))
     }
 
+    private fun printPrize(
+        prize: Prize,
+        count: Int,
+    ) {
+        if (prize == Prize.SECOND) {
+            println("${prize.matchCount} Matches + Bonus Ball (${prize.value.toMoneyExpression()} KRW) - $count tickets")
+            return
+        }
+        println("${prize.matchCount} Matches (${prize.value.toMoneyExpression()} KRW) - $count tickets")
+    }
```

Comment:
Why not take the chance to learn and use the `buildString` API and remove duplicates? ☺️ 

https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.text/build-string.html

Reply:
I was thinking `${if (prize == SECOND) ... }` to remove duplicates but this looks like a cool DSL feature so I tried 😆